docs/photutils/psf.rst

```diff
@@ -481,6 +481,159 @@ Consider the previous example after the line
     plt.colorbar(orientation='horizontal', fraction=0.046, pad=0.04)
     plt.show()
 
+Fitting additional parameters
+-----------------------------
+
+The PSF photometry classes can also be used to fit additional parameters
+simultaneously, for instance, the PSF width and the sky background.
```

Comment:
This particular use case is at first glance a bit odd.  One of the main points of PSF photometry is to use the *shape* information to get a better constraint on the flux and centering. So it would be odd to want to independently measure each sigma on its own.  The "sky background" use case is much more practical, but that's a bit more complicated to implement because there's not a natural model to use.

So maybe something like this:  
```
... to fit more model parameters than just the flux and centering.  
While a more realistic use case might be fitting sky backgrounds, 
or shape parameters of galaxies, here we use the ``sigma`` 
parameter in `IntegratedGaussianPRF` as the simplest possible
example of this feature.
```

Alternatively, you could adapt the example to use the astropy `Gaussian2D`, and have x_stddev/y_stddev/theta *all* be free parameters.  That would then sort of be like galaxy-fitting photometry. But that might get a bit overly-complex, so whichever you think is going to be clearer is fine, @mirca